pkg/virt-handler/heartbeat/multipath-socket_test.go

```diff
@@ -0,0 +1,132 @@
+package heartbeat
```

Comment:
```suggestion
package heartbeat_test
```

Should be a test package?